libcloud/common/dimensiondata.py

```diff
@@ -806,12 +806,15 @@ class DimensionDataFirewallAddress(object):
     The source or destination model in a firewall rule
     """
     def __init__(self, any_ip, ip_address, ip_prefix_size,
-                 port_begin, port_end):
```

Comment:
Just a heads up - looks like this change is backward incompatible since it broke st2contrib action (pack there is using latest trunk) - https://travis-ci.org/StackStorm/st2contrib/jobs/125555233#L455 :)

Seems like the solution would be to default new constructor arguments to `None` if they are optional and / or update the pack action to supply those arguments.